osquery/tables/system/hash.cpp

```diff
@@ -195,6 +197,12 @@ bool FileHashCache::load(const std::string& path,
 
 Status genSsdeepForFile(const std::string& path, std::string& ssdeep_hash) {
 #ifdef OSQUERY_POSIX
+  boost::filesystem::path p = path;
+  auto size = boost::filesystem::file_size(p);
```

Comment:
Make sure to use the overload where you can retrieve the error_code (see the other usages of `boost::filesystem` in this file), otherwise boost may throw an exception.

Reply:
nice catch, thanks @theopolis